src/test/java/core/basesyntax/service/FruitReportServiceImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.impl.FruitReportServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FruitReportServiceImplTest {
+
+    private static final String BANANA_FRUIT = "banana";
+    private static final String APPLE_FRUIT = "apple";
+    private static final int APPLE_QTY = 20;
+    private static final int BANANA_QTY = 30;
+    private FruitReportService reportService;
+
+    @BeforeEach
+    void setUp() {
+        reportService = new FruitReportServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.storage.clear();
+    }
+
+    @Test
+    void createReport_normalCase_Ok() {
```

Comment:
What is normal case? Maybe you shoud name it 
```suggestion
    void createReport_createWithValidData_Ok() {
```